shell/platform/darwin/common/buffer_conversions.h

```diff
@@ -10,16 +10,17 @@
 #include <vector>
 
 #include "flutter/fml/mapping.h"
+#import "flutter/lib/ui/window/platform_message.h"
```

Comment:
This could be removed if my other PR lands where I migrate PlatformMessage to std::unique_ptr, then I could steal it's std::vector instead of wrapping the PlatformMessage.